packages/node_modules/@node-red/editor-client/src/js/ui/view.js

```diff
@@ -5584,7 +5494,23 @@ RED.view = (function() {
             redraw(true);
         },
         selection: getSelection,
-
+        clearSelection: clearSelection,
+        get selectedLinks() {
```

Comment:
This should not be exposed - it is internal editor state.

`RED.view.selection().links` gives the same information, but as a cloned array so it can't be modified (accidentally or otherwise).

Reply:
I missed that one! Done :)